src/app/_components/HaveAnAccount/HaveAnAccount.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { Box } from "@mui/material";
+import { HaveAnAccount } from "./HaveAnAccount";
+
+const meta: Meta = {
+  title: "Authentication/HaveAnAccount",
+  component: HaveAnAccount,
+  parameters: {
+    layout: "centered",
+  },
+  decorators: [
+    (Story) => (
+      <Box>
+        <Story />
+      </Box>
+    ),
+  ],
+} satisfies Meta<typeof HaveAnAccount>;
+
+export default meta;
+type Story = StoryObj<typeof meta>;
+
+export const Default: Story = {
+  args: {},
```

Comment:
your story args doesm't take required props, onLogin and onTextClick, but that maybe okay since it just your story but everything else looks correct